### DISCUSSION TITLE
On the alternate allele frequency

### DISCUSSION THREAD

**Discussion by jinshanmu** [MRCIEU/TwoSampleMR#429] [Q&A] [answered]
I have a question on the number of alternate allele frequency (AF) / effect allele frequency (EAF) in the IEU OpenGWAS. Take rs12562034 in the ieu-a-1013 data for example, it has AF = 0.895 for the alternate allele A. However, when querying PhenoScanner or GRCh37 Ensembl, the A allele has a frequency of 0.09 in the European population. Is there a confusion between alternate allele and reference allele in the data curation process?

**Comment by mightyphil2000** [comment, marked as answer]
Hi Jinshanmu

Thanks for bringing this to our attention.

There is a metadata error in dataset ieu-a-1013. The effect allele columns are mislabelled. The effect allele (ea) is actually the non-effect allele (nea) and effect allele frequency (eaf) actually refers to the non-effect allele (nea).
________________________________
From: jinshanmu ***@***.***>
Sent: 16 July 2023 03:36
To: MRCIEU/TwoSampleMR ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [MRCIEU/TwoSampleMR] On the alternate allele frequency (Discussion #429)


I have a question on the number of alternate allele frequency (AF) / effect allele frequency (EAF) in the IEU OpenGWAS. Take rs12562034 in the ieu-a-1013 data for example, it has AF = 0.895 for the alternate allele A. However, when querying PhenoScanner or GRCh37 Ensembl, the A allele has a frequency of 0.09 in the European population. Is there a confusion between alternate allele and reference allele in the data curation process?

**Reply by jinshanmu** [reply]
Thank you for the clarification!
